pbl

```diff
@@ -38,25 +40,51 @@ my $loader;
 my $exit_code = 0;
 my @todo;
 
+my $opt_logfile = "/var/log/pbl.log";
+
+
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# pbl_usage($exit_code)
+#
+# Print help text and exit.
+#
+sub pbl_usage
+{
+  print <<"= = = = = = = =";
+Usage: pbl [OPTIONS]
+Configure/install boot loader.
+
+Options:
+    --install               Install boot loader.
+    --config                Create boot loader config.
+    --show                  Print current boot loader name.
+    --default ENTRY         Set default boot entry to ENTRY.
+    --add-option OPTION     Add OPTION to default boot options.
+    --del-option OPTION     Delete OPTION from default boot options.
```

Comment:
We can think up something when we need it. I don't want to blow up the code just yet. I think
the design is flexible enough at this point to allow extensions easily.